src/Models/UsersReactions.php

```diff
@@ -38,4 +49,16 @@ public function initialize()
 
         $this->setSource('users_reactions');
     }
+
+
+    /**
+     * afterSave.
+     *
+     * @return void
+     */
+    public function afterSave()
+    {
+        $users = Users::findFirstOrFail($this->users_id);
+        Interactions::add($users, $this->entity_namespace::findFirst($this->entity_id), UsersInteractions::REACTION);
```

Comment:
@FredPeal encapsulate this metho, you are using it in multiple  location
$this->entity_namespace::findFirst($this->entity_id)